docs/using-the-compiler.rst

```diff
@@ -201,14 +201,23 @@ Input Description
           }
         }
         // The following can be used to select desired outputs.
-        // If this field is omitted, then the compiler loads and does type checking, but will not generate any outputs apart from errors.
-        // The first level key is the file name and the second is the contract name, where empty contract name refers to the file itself,
+        // If this field is omitted, then the compiler loads and does type checking,
+        // but will not generate any outputs apart from errors.
+        // The first level key is the file name and the second is the contract name,
+        // where empty contract name refers to the file itself,
         // while the star refers to all of the contracts.
+        // To get everything the compiler can possibly generate, use
```

Comment:
"To select all supported kinds of output for all contracts"?
Probably fine as is, though.

Reply:
"To select everything the compiler can possibly generate"?

Reply:
all outputs the compiler can possibly generate?

Reply:
"To select all outputs the compiler can possibly generate" sounds best to me.